File: OLD_JULIA/src_histogram/remove_missing_rows.jl
#=
    Removes the entire row if one value is missing
=#
function remove_missing_rows(numerical_values, house_values, numerical_headers)
    i = 0
    println(length(house_values))
    for elem in 0:length(house_values)
        println(i)
        i += 1;
    end
    println("i = $i")
end

File: src_scatter_plot/remove_missing_rows.py
from src_histogram.get_num_headers import get_num_headers
"""
    Removes the entire row when one numerical value is missing
    ==> Returns the new table
"""
def remove_missing_rows(values):
    for head in values:
        i = len(values[head]) - 1
        while i >= 0 :
            if (values[head][i] == ""):
                for head in values:
                    values[head].pop(i)
            else:
                i -= 1
    return (values)
